$MirOS: ports/www/dillo/v2/patches/patch-dpid_dpid_common_h,v 1.1 2008/11/07 18:05:52 bsiegert Exp $
--- dpid/dpid_common.h.orig	Mon Jan 18 18:00:19 2010
+++ dpid/dpid_common.h	Sun Feb 21 18:23:26 2010
@@ -21,9 +21,9 @@
 #define _MSG_ERR(...)
 #define MSG_ERR(...)  fprintf(stderr, "[dpid]: " __VA_ARGS__)
 
-#define dotDILLO_DPI ".dillo/dpi"
-#define dotDILLO_DPIDRC ".dillo/dpidrc"
-#define dotDILLO_DPID_COMM_KEYS ".dillo/dpid_comm_keys"
+#define dotDILLO_DPI ".etc/dillo/dpi"
+#define dotDILLO_DPIDRC ".etc/dillo/dpidrc"
+#define dotDILLO_DPID_COMM_KEYS ".etc/dillo/dpid_comm_keys"
 
 #define ERRMSG(CALLER, CALLED, ERR)\
  errmsg(CALLER, CALLED, ERR, __FILE__, __LINE__)
